Library/OptimizelyClientType.swift

```diff
@@ -36,6 +36,31 @@ extension OptimizelyClientType {
 
     return variant
   }
+
+  /*
+   Calls `getVariation` on the Optimizely SDK for the given experiment,
+   using the default attributes and deviceId
+
+   Does *not* record an Optimizely impression. If you wish to record an experiment impression, use
+   `variant(for experiment)`, which calls `activate` on the Optimizely SDK
+   */
+
+  public func getVariation(for experiment: OptimizelyExperiment.Key) -> OptimizelyExperiment.Variant {
```

Comment:
The docs are unclear on whether `activate` is idempotent or not 🤔 
> Use Activate when the visitor actually sees the experiment. Use Get Variation when you need to know which bucket a visitor is in before showing the visitor the experiment.

This seems to suggest calling `getVariation` _before_ `activate` but I understand that in your scenario you are using it because the experiment is _already_ activated.

If it _is_ idempotent, do we care?

Reply:
I think this change is mainly a safety measure to minimize the chances of getting double-charged for impressions. In terms of our actual experiment, calling `activate` instead of `getVariation` would not affect the results, because like you said it's already been activated during the onboarding flow.